cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-simple-cli-guide.adoc

```diff
@@ -0,0 +1,45 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-getting-started-simple-cli-guide"]
+= Tutorial: Simple CLI guide
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-getting-started-simple-cli-guide
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2023-11-16
+
+This page outlines the minimum list of commands to deploy a ROSA cluster using the command line interface (CLI). 
+
+[NOTE]
+====
+While this simple deployment works well for a workshop setting, clusters used in production should be deployed with a more detailed method.
+====
+
+== Prerequisites
+
+//* You have completed the prerequisites in the *xref needed*[Setup] section.
+
+== Creating account roles
+Run the following command _once_ for each AWS account and y-stream OpenShift version:
+
+[source,terminal]
+----
+rosa create account-roles --mode auto --yes
+----
+
+== Deploying the cluster
+
+. Create the cluster with the default configuration by running the following command substituting your own cluster name:
```

Comment:
Do we need to point out that in the example given, this is specifically an STS cluster? Not sure how much detail there needs to be in these tutorials? 


```suggestion
. Create the cluster with the default configuration by running the following command, substituting your own cluster name:
```

Reply:
Yes, you are totally correct. One of the previous tutorials in this "workshop" says this is an STS-cluster. We assume they will follow the entire workshop, so no need to repeat STS.